.github/workflows/production-performance.yml

```
@@ -0,0 +1,98 @@
+
+concurrency:
+  group: production-performance
+  cancel-in-progress: false
+
+jobs:
+  measure:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    env:
+      PROD_BASE_URL: https://comic-pile.vercel.app
+      PROD_PROFILE_STORAGE_STATE: ${{ runner.temp }}/production-storage-state.json
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)